modules/custom/az_core/config/quickstart/search.page.node_search.yml

```diff
@@ -1,12 +1 @@
-langcode: en
-status: false
-dependencies:
-  module:
-    - node
-id: node_search
-label: 'Content (old)'
-path: default
-weight: -10
-plugin: node_search
-configuration:
-  rankings: {  }
+{  }
```

Comment:
Is it intended that this file should be deleted instead?

Reply:
Definitely seems like deleting the file would be preferable.